style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  line-height: 1.5;
+  font-size: 16px;
+  background-color: rgb(235, 235, 235);
+}
+
+img {
+  padding: 25px 0;
+  width: 25%;
+}
+
+header {
+  display: flex;
```

Comment:
Great use of the flex display 👏 . However, `display: flex` is being repeated in multiple CSS blocks. To make the code dry, it'll be nice to write this display once for all the elements, ids and classes that it's being used for.
It should look like this 👇 

```` 
     header,
     #features h2,
     #pricing {
     display: flex
     }

Reply:
Remember to leave a polite comment with the student!

And even though you are pointing out a mistake in the code you are not explaining why you requested the change.
In this case, I think you should point out that the code should be free from repetition and be as clean as possible. Please check out the `DRY`, `KISS`, and `YAGNI` [principles.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md)

Reply:
Remember to leave a polite comment with the student!
And even though you are pointing out a mistake in the code you are not explaining why you requested the change.

**In this case, I think you should point out that the code should be free from repetition and be as clean as possible. Please check out the DRY, KISS, and YAGNI principle**